server/auth.js

```diff
@@ -112,14 +112,18 @@ auth.post('/signup/local', (req, res, next) =>
   User.findOne({
     where: {
       email: req.body.email
-    }
+    },
+    attributes: {include: ['password_digest']}
   })
   .then(user => {
-    if (!user) User.create(req.body)
-    return user
+    if (!user) return User.create(req.body)
+    else return user.authenticate(req.body.password)
```

Comment:
For fun, an alternate format:

```js
.then(user =>  user
  ? User.create(req.body)
  : user.authenticate(req.body.password)
)
```

No need to actually use this, just showing alternatives.